### DISCUSSION TITLE
Mainline build with 6.8 rc2

### DISCUSSION THREAD

**Discussion by stevepringle** [Joshua-Riek/ubuntu-rockchip#607] [General]
Hello, just wondering if there was any chance of a mainline build based on 6.8 rc2 as there are supposed to be further additions to support to rk3588 from Colabora included in the 6.8 kernel.

**Comment by thewonderer73** [comment]
https://gitlab.collabora.com/hardware-enablement/rockchip-3588/notes-for-rockchip-3588/-/blob/main/mainline-status.md

**Reply by Deiiji** [reply]
Hello, you have 6.8rc2 in some armbian nightly, but is not yet stable
